File: src/C#/CreationalPatterns/AbstractFactory/EraStyles/Products/RenaissanceHouse.cs
using EraStyles.Abstraction;
using EraStyles.Constants;
using System;
using System.Collections.Generic;
using System.Text;

namespace EraStyles.Products
{
    public class RenaissanceHouse : EraObject
    {
        public override void ShowDetails()
        {
            Console.WriteLine(string.Format(GlobalConstants.HouseMessage, "Renaissance"));
        }
    }
}

File: src/JavaScript/CreationalPatterns/AbstractFactory/EraStyles/products/renaissanceHouse.js

const EraObject = require("../abstraction/eraObject");
const GlobalConstants = require("../constants/globalConstants");

class RenaissanceHouse extends EraObject {
    showDetails() {
        console.log(GlobalConstants.HouseMessage.replace("{0}", "Renaissance"));
    }
}

module.exports = RenaissanceHouse;